src/app/tests/suites/certification/Test_TC_OVENOPSTATE_2_5.yaml

```diff
@@ -0,0 +1,263 @@
+# Copyright (c) 2023 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Auto-generated scripts for harness use only, please review before automation. The endpoints and cluster names are currently set to default
+
+name: 204.2.5. [TC-OVENOPSTATE-2.5] Optional Events with DUT as Server
```

Comment:
@beckerzito I could see there is no test case named TC-OVENOPSTATE-2.5 in test plan 

Ref : https://github.com/CHIP-Specifications/chip-test-plans/blob/master/src/cluster/operationalstatecommon.adoc

![image](https://github.com/project-chip/connectedhomeip/assets/95559513/3ce74b04-6163-4496-9edd-ffdff807c28c)

Reply:
hi @Survensa thanks for that! You're right. Actually we are working in advance by adding this PR https://github.com/CHIP-Specifications/chip-test-plans/pull/3844. I'm gonna add that to the comments of the PR